### ISSUE TITLE
US: Translation of the state Georgia collides with the country

### ISSUE THREAD

**Issue by lukasklein** [django/django-localflavor#249]
Turns out there's an US state Georgia and a country with both the exact same name, but, at least in Germany, only the country Georgia has a translation. So translating the US state to the same thing as the country causes confusion.
I have "fixed" this in our project using

``` python
US_STATES_GEORGIAFIX = map(lambda state: ('GA', "Georgia") if state[0] == 'GA' else state, US_STATES)
```

More generally asking: Should US state names be translated at all?

**Comment by claudep** [comment]
> More generally asking: Should US state names be translated at all?

Yes, absolutely.

The solution to disambiguation is to use pgettext_lazy (`('GA', pgettext_lazy('US state', 'Georgia')`).

**Comment by benkonrath** [comment]
This has been fixed but the translations probably need to be updated before it will be fully solved.

**Comment by claudep** [comment]
The translation catalog has been updated in https://github.com/django/django-localflavor/commit/ca33c5b864dd57fc3ccd50d709ede3cd81982b7e. Transifex should contain the modified strings soon…

**Comment by benkonrath** [comment]
A fix for this issue was included in the 1.4 release.
